SemesterReview.tex

```diff
@@ -0,0 +1,38 @@
+\documentclass{scrartcl}
+
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+\usepackage{amsmath}
+
+\title{Semester Review Report}
+
+
+
+
+\author{DH185421}
+
+\begin{document}
+
+\maketitle
+
+\abstract{This report will be looking at the skills that I have personally been lacking and what next steps I should take towards improving them.}
+
+
+
+
+
+\section{Key Skill 1 - Communication}
+
+Communication is a key part for a student to use to be able to get and give information regarding the course and work set. For me the most significant aspect was to be able to know what I am doing and know where I am going wrong. Alternatively this is perhaps due to some sort of anxiety I get when asking for help as I don’t want to burden anyone with my problems. Equally this is probably shown by the lack of quality and quantity of my work in the first semester. Having experienced not knowing what to do until it's nearly too late I now feel like I need to ask for help when I need it, additionally I have learned that I need to develop a rapport with my lecturers and classmates so I am not afraid to ask. I have slightly developed my skills in communication by helping and talking about the work with a classmate. This skill will be essential to me as a learner and a practitioner because without it I will not fully understand what is expected of me/work. As a next step I will need to communicate more with my lecturer to make sure I am on task.
+
+\section{Key Skill 2 - Time Management}
+Time management is a key skill especially in programming and writing essays as it cannot be crammed into a single night. For me the most relevant issue was not properly organising my time on task. Subsequently, I noticed that I had started to fall behind on work. Equally this might be due to family matters. Having experienced that having to cram work in under a week is stressful I now feel like I will spend my time more productively. Furthermore, I have learned that even doing a little bit of work each day adds up. I have significantly improved my ability to time manage by doing a little of work each day or at least every other day. This makes me feel more in control of my work is alot less stressful than having to do it all a week before hand in. As a next step I will need to write out a plan of what work I need to be doing and plan my time accordingly.
```

Comment:
I would suggest liaising with the [ASK Team](http://ask.fxplus.ac.uk/) to see what specific actions you could attempts. The proposed action of 'plan my time accordingly' is too general to be SMART --- *Specific, Measurable, Achievable, Relevant, and Time Bound*